src/code.c

```diff
@@ -2257,12 +2257,7 @@ void CodeAssLVar (
     Expr                rhsx;           /* right hand side expression      */
 
     /* allocate the assignment                                             */
-    if ( lvar <= 16 ) {
-        ass = NewStat( T_ASS_LVAR + lvar, 2 * sizeof(Stat) );
-    }
-    else {
-        ass = NewStat( T_ASS_LVAR,        2 * sizeof(Stat) );
-    }
```

Comment:
I also checked I can't get this to go and faster. it's not suprising, as you the T_ASS_LVAR_xx aren't even any smaller, they still contain the lvar value, so all you are saving is reading one of two values which are right next to each other, which are almost certainly in the same cache line so the CPU already has them both anyway.